modules/ocf_stats/files/munin/munin.conf

```diff
@@ -92,7 +92,7 @@ html_strategy cgi
 #
 # For those with Nagios, the following might come in handy. In addition,
 # the services must be defined in the Nagios server as well.
-#contact.nagios.command /usr/bin/send_nsca nagios.host.comm -c /etc/nsca.conf
+contact.root.command | /usr/local/bin/mail-munin-alert /usr/local/bin/mail-munin-alert ${var:host} ${var:wfields} ${var:cfields}
```

Comment:
FYI I tried removing the superfluous first argument, and Munin stopped sending any arguments at all. The way it appears here is consistent with the [documentation](http://guide.munin-monitoring.org/en/latest/tutorial/alert.html#alerts-to-or-through-external-scripts). I'll look for a fix if one exists.